Add documentation clarification (#206)
dapr/js-sdk: daprdocs/content/en/js-sdk-docs/_index.md

The Dapr JS SDK will allow you to interface with the Dapr process that abstracts several commonly used functionalities such as Service-to-Service invocation, State Management, PubSub, and more.

## Installation

> To get started with the Javascript SDK, you can download the Dapr Javascript SDK package from [NPM](https://npmjs.org/package/dapr-client) by running the below:

Reviewer: ```suggestion
To get started with the Javascript SDK, you can download the Dapr Javascript SDK package from [NPM](https://npmjs.org/package/dapr-client) by running the following:
```

```bash
npm install --save dapr-client
```

## Structure

> The Dapr Javascript SDK exists out of 2 major components: 

Reviewer: ```suggestion
The Dapr Javascript SDK contains two major components: 
```

* **DaprServer:** The Dapr Server manages all communication from the Dapr Sidecar to your application
* **DaprClient:** The Dapr Client manages all communication from your application to the Dapr Sidecar

> To achieve communication, you are able to choose between 2 different protocols that are implemented: gRPC or HTTP 

Reviewer: ```suggestion
The above communication can be configured to use either of the gRPC or HTTP protocols.
```

> ![Dapr Server](./js-server/dapr-server.jpg)

Reviewer: The images look slightly oversized in the preview, do you want to constraint it to a specific size? https://github.com/dapr/js-sdk/blob/aa14ebf406fb0ac376d6306971503a244011d771/daprdocs/content/en/js-sdk-docs/_index.md

> ![](./js-server/dapr-server.jpg)

Reviewer: nit: alt text for images

![](./js-client/dapr-client.jpg)

## Get Started

> To help you get started, feel free to check out the resources below:

Reviewer: ```suggestion
To help you get started, check out the resources below:
```

<div class="card-deck">
  <div class="card">
    <div class="card-body">
      <h5 class="card-title"><b>Client</b></h5>

>       <p class="card-text">Create a JavaScript client and interact with a Dapr sidecar and other Dapr applications. (e.g. Publishing events, Output Binding support, ...)</p>

Reviewer: ```suggestion
      <p class="card-text"> Create a JavaScript client and interact with a Dapr sidecar and other Dapr applications. (e.g., publishing events, output binding support, etc.)</p>
```

      <a href="{{< ref js-client >}}" class="stretched-link"></a>
    </div>
  </div>
  <div class="card">
    <div class="card-body">
      <h5 class="card-title"><b>Server</b></h5>

>       <p class="card-text">Create a JavaScript server and let the Dapr sidecar interact with your application. (e.g. Subscribing to events, Input Binding support, ...) </p>

Reviewer: ```suggestion
      <p class="card-text"> Create a JavaScript server and let the Dapr sidecar interact with your application. (e.g., subscribing to events, input binding support, etc.) </p>
```
